tileview/src/main/java/com/qozix/tileview/markers/MarkerLayout.java

```diff
@@ -268,6 +302,16 @@ public LayoutParams( int width, int height, int left, int top, Float anchorLeft,
       anchorY = anchorTop;
     }
 
+    public LayoutParams( int width, int height, int left, int top, Float anchorLeft, Float anchorTop, int typeAnchorX, int typeAnchorY ) {
+      super( width, height );
+      x = left;
+      y = top;
+      anchorX = anchorLeft;
+      anchorY = anchorTop;
+      this.typeAnchorX = typeAnchorX;
+      this.typeAnchorY = typeAnchorY;
```

Comment:
no `this` on the above 2 lines.  when the member names are updated to include the `m` prefix, it will naturally disambiguate

Reply:
I have renamed the variables so this is no longer needed.